### PR TITLE
CLDC-1179: cya change link text

### PR DESCRIPTION
**Pull request by kiddhustle**
https://digital.dclg.gov.uk/jira/browse/CLDC-1179

### CODE REVIEW

**Comment by paulrobertlloyd** [comment]
@kiddhustle I think I addressed the issues DAC saw by using the summary list component in #499, so this should no longer be required.

**Comment by kiddhustle** [comment]
@paulrobertlloyd did your fix address the organisation page also?

**Comment by paulrobertlloyd** [comment]
Good question! I fixed the spacing issue, but the link text needs updating still, as they all say ‘Change name’

Screenshot 2022-04-27 at 11 27 45

**Comment by kiddhustle** [comment]
Screenshot from 2022-04-27 11-34-01

**Comment by kiddhustle** [comment]
Screenshot from 2022-04-27 11-41-24

@paulrobertlloyd  The text has been amended in this PR but only for the organisation. I think the section CYA are handled differently

File: app/views/organisations/show.html.erb [hunk 1]
```diff
@@ -14,7 +14,7 @@
             <% row.key { attr[:name].to_s.humanize } %>
             <% row.value { details_html(attr) } %>
             <% row.action(
-              visually_hidden_text: "name",
+              visually_hidden_text: "&nbsp;" + attr[:name].to_s.humanize.downcase,
```
Comment (paulrobertlloyd) [inline app/views/organisations/show.html.erb]: Ahhhh sorry, I’m getting my different summary lists mixed up. In which case, I don’t think the `&nbsp;` is required here: 

```suggestion
              visually_hidden_text: attr[:name].to_s.humanize.downcase,
```

Comment (kiddhustle) [reply app/views/organisations/show.html.erb]: yep just removed that now

Comment (paulrobertlloyd) [reply app/views/organisations/show.html.erb]: Brill, thank-you 🙂

**Review by baarkerlounger — APPROVED** [review]

**Pull request merged**